backend/core/Extensions/StringExtensions.cs

```diff
@@ -78,5 +78,19 @@ public static HttpMethod GetHttpMethod(this string method)
                 _ => HttpMethod.Post,
             };
         }
+
+        /// <summary>
+        /// Convert the specified 'value' from default to UTF8 encoding.
+        /// Replace linebreaks with spaces.
+        /// </summary>
+        /// <param name="value"></param>
+        /// <param name="replaceLineBreaks"></param>
+        /// <returns></returns>
+        public static string ConvertToUTF8(this string value, bool replaceLineBreaks = true)
```

Comment:
Without this the db was getting populated with \n newline characters.  Which causes issues when searching.

Reply:
So this extension gets called without us having to call it explicitly?